sgp/Video.cc

```diff
@@ -142,27 +124,64 @@ void InitializeVideoManager(void)
 {
 	DebugMsg(TOPIC_VIDEO, DBG_LEVEL_0, "Initializing the video manager");
 
-	SDL_WM_SetCaption(APPLICATION_NAME, NULL);
-  ClippingRect.set(0, 0, SCREEN_WIDTH, SCREEN_HEIGHT);
+	g_game_window = SDL_CreateWindow(APPLICATION_NAME,
+                              SDL_WINDOWPOS_UNDEFINED,
+                              SDL_WINDOWPOS_UNDEFINED,
+                              SCREEN_WIDTH, SCREEN_HEIGHT,
+                              g_window_flags);
+    GameRenderer = SDL_CreateRenderer(g_game_window, -1, 0);
+    SDL_RenderSetLogicalSize(GameRenderer, SCREEN_WIDTH, SCREEN_HEIGHT);
+
+
+    ClippingRect.set(0, 0, SCREEN_WIDTH, SCREEN_HEIGHT);
+
+    Uint32 Rmask = 0xF800;
+    Uint32 Gmask = 0x07E0;
+    Uint32 Bmask = 0x001F;
+    Uint32 Amask = 0;
+	ScreenBuffer = SDL_CreateRGBSurface(
+	           0,
+	           SCREEN_WIDTH,
+	           SCREEN_HEIGHT,
+	           PIXEL_DEPTH,
+               Rmask,
+               Gmask,
+               Bmask,
+               Amask
+    );
+
+    if (ScreenBuffer == NULL) {
+        SLOGD("Topic", "SDL_CreateRGBSurface for ScreenBuffer failed: %s\n", SDL_GetError())
```

Comment:
i see that a ';' is not neccessary here, but every other place in the codebase puts semicolons there.
The #define for SLOGD adds it, but i would prefer consistency across the code